### PR TITLE
Update https for better clarity and resiliance

### LINKED ISSUE #4: Fix .key permissions for certbot

### LINKED ISSUE #5: Update certbot docs to show the need for sudo

### LINKED ISSUE #6: Stop process on error for certbot

### LINKED ISSUE #12: have https create the .key folder

### PR DESCRIPTION
**Pull request by sirmspencer**
close #4 
close #5 
close #6 
close #12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 .cpcache/
 .nrepl-port
-pom.xml
-target/
+.rebel_readline_history
```
Comment (rjsheperd) [inline .gitignore:3]: I would prefer to keep `pom.xml` and `target` in the git-ignore

**Review by rjsheperd — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (sirmspencer) [reply .gitignore:3]: What do you expect to create pom.xml or target?

Comment (rjsheperd) [reply .gitignore:3]: `target` and `pom.xml` files usually created when building a JAR / UberJAR. Those don't need to be committed.

**Review by rjsheperd — APPROVED** [review]

**Comment by Kcheung42** [comment]
triangulum.https
- line 41: end string without space
- line 42-43: begin string with space
- line 43: end string with space

**Pull request merged**